templates/workspace_services/guacamole/terraform/web_app.tf

```diff
@@ -35,15 +35,15 @@ resource "azurerm_app_service" "guacamole" {
     GUAC_DRIVE_NAME       = "${var.guac_drive_name}"
     GUAC_DRIVE_PATH       = "${var.guac_drive_path}"
     GUAC_DISABLE_DOWNLOAD = "${var.guac_disable_download}"
-    AUDIENCE              = "${var.openid_client_id}"
+    AUDIENCE              = "${var.ws_client_id}"
     ISSUER                = local.issuer
 
-    OPENID_AUTHORIZATION_ENDPOINT = "https://login.microsoftonline.com/${local.aad_tenant_id}/oauth2/v2.0/authorize"
-    OPENID_JWKS_ENDPOINT          = "https://login.microsoftonline.com/${local.aad_tenant_id}/discovery/v2.0/keys"
-    OPENID_ISSUER                 = "https://login.microsoftonline.com/${local.aad_tenant_id}/v2.0"
-    OPENID_CLIENT_ID              = "${var.openid_client_id}"
-    OPENID_REDIRECT_URI           = "https://${local.webapp_name}.azurewebsites.net/guacamole/"
-    OPENID_USERNAME_CLAIM_TYPE    = "email"
+    OAUTH2_PROXY_CLIENT_ID       = "${var.ws_client_id}"
+    OAUTH2_PROXY_CLIENT_SECRET   = "${var.ws_client_secret}"
```

Comment:
Would be nicer if this was stored in KeyVault as part of the workspace deployment and the KV secret referenced. Maybe one to pick up in a separate issue?

Reply:
This is the workspace_app_id and the workspace_app_secret that are being transferred from the .env file. So it is not created as part of the workspace deployment...

Reply:
Do you think it should be part of this PR scope?

Reply:
https://github.com/microsoft/AzureTRE/issues/1693
Feel free to change